tests/util_test.py

```diff
@@ -141,3 +141,40 @@ def simple_name_marker(contact, name, soup):
         assert out == util.autolink(
             inp, person_processor=simple_name_marker,
             url_processor=None)
+
+
+def test_parsing_hashtags():
+    """Exercise the #-tag matching regex
+    """
+
+    test_cases = [
+        ('#hashtag should be linked', 
+         '<a href="/tags/hashtag">#hashtag</a> should be linked',
+         ['hashtag']),
```

Comment:
heh, I wonder if this will fix the problem where markdown thinks a # at the beginning of a note means it should be a header.